src/container/cli.c

```diff
@@ -1868,15 +1869,22 @@ cont_req_cleanup(enum creq_cleanup_stage stage, struct cont_req_arg *args)
 	switch (stage) {
 	case CLEANUP_ALL:
 		crt_req_decref(args->cra_rpc);
+		break;
```

Comment:
It's amazing this code worked at all!

Reply:
yeah, multiple places like this. I bluntly add break in those places with the same question + "could it be intentional fall through?" which no signs of that.